lib/omnibus/fetchers/git_fetcher.rb

```diff
@@ -203,7 +203,7 @@ def contains_revision?(rev)
     #   the shellout object
     #
     def git(command)
-      shellout!("git #{command}", cwd: project_dir)
+      shellout!("git -c core.autocrlf=false #{command}", cwd: project_dir)
```

Comment:
Yeah I can add a comment to that.  autocrlf is a complete and utter hack (and so are it's bretheren in .gitattributes).  IMHO, a version control system should never be "guessing" the semantics of your file and modifying the syntax to suit.  That's the job of the compiler or IDE or whatever higher level tool whose job is to interpret your typing-thoughts.  autocrlf is a hack to allow a "canonical" representation of a file inside the underlying git repository while seamlessly checking it out with different line endings on different OSes.  That's what caused our build issue here - a certain config.hpp.in was being checked out on windows with \r\n line endings which makes an awk script not correctly match the line.  I figure that the source should look the same on whichever platform we build and if there is a translation needed, we should manually ask for it in the software definition (and this should happen regardless of whether we're on a build machine or on a developer build VM).